WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -59,8 +59,10 @@
           <% else %>
             You have registered for <%= @competition.name %>.
             You can see your registration information below.
-            <%# TODO - see https://github.com/cubing/worldcubeassociation.org/issues/269 %>
-            Contact the organizer if you wish to make a change.
+            
+            <% if !current_user.can_edit_registration?(@registration) %>
```

Comment:
This is called in several places. I'd compute **@editable_registration** in the controller.

Also, I prefer **unless* to **if !**.